packages/element/src/utils.ts

```
@@ -1,28 +1,168 @@
+    if (version !== element.version) {
+      ElementShapeCache.cache.set(element, {
+        version: element.version,
+        shapes: new Map([[offset, shape]]),
+      });
+
+      return;
+    }
+
+    shapes.set(offset, shape);
+  };
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)